src/engine/enginechannel.h

```diff
@@ -39,7 +39,8 @@ class EngineChannel : public EngineObject {
     };
 
     EngineChannel(const ChannelHandleAndGroup& handle_group,
-                  ChannelOrientation defaultOrientation = CENTER);
+                  ChannelOrientation defaultOrientation = CENTER,
+                  bool isInput = false);
```

Comment:
I do not understand why those distinctions are necessary. But let's keep this PR focused on the new changes. We can discuss the possibility of consolidating the mic and aux inputs after merging this.